### ISSUE TITLE
Windows 10: wails init does not work with Git Bash

### ISSUE THREAD

**Issue by pedzed** [wailsapp/wails#166]
**Description**
When running `wails init` with [Git Bash](https://git-scm.com/downloads) (running on Windows 10), pressing the enter key does not work anymore after "Please select a template:". Running `wails init` with cmd.exe works, but it is not ideal.

![image](https://user-images.githubusercontent.com/8146593/61774008-499b8600-adf6-11e9-8424-c4623587da1f.png)

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v0.17.0 |
| Go Version    | go1.12.7 |
| Platform      | windows |
| Arch          | amd64 |
| GO111MODULE   | (Not Set) |

**Additional context**
Go and NodeJS installed using [Scoop](https://scoop.sh/):
```bash
scoop install go
scoop install nodejs
```

**Comment by leaanthony** [comment]
Thanks for opening this. There's a batch of Windows testing that will be happening this weekend so I'll include this and your other ticket in that.

**Comment by leaanthony** [comment]
I think I have a fairly trivial fix pushed to the latest pre-release for you. Please update to `v0.17.9-pre` using `wails update -pre` and test again. 

Let us know how you get on 👍

**Comment by pedzed** [comment]
Excellent!

```bash
$ wails init
Wails v0.17.9-pre - Initialising project

The name of the project (My Project): x
Project Name: x
The output binary name (x):
Output binary Name: x.exe
Project directory name (x):
Project Directory: x
Please select a template:
  1: Angular - Angular 8 template (Requires node 10.8+)
  2: React JS - Create React App v3 template
  3: Vue2/Webpack Basic - A basic Vue2/WebPack4 template
  4: Vuetify Basic - Vuetify + Webpack
Please choose an option [1]: 3
Template: Vue2/Webpack Basic
> Generating project...
> Building project (this may take a while)...
Project 'x' built in directory 'x'!
```

**Comment by pedzed** [comment]
Unfortunately it seems that this issue has a regression in `Wails v0.17.0`.

**Comment by leaanthony** [comment]
Thanks for letting us know @pedzed - I'll look into it.

**Comment by pedzed** [comment]
Never mind, apparently I forgot to upgrade to the RC release (on a new setup).

**Comment by leaanthony** [comment]
Ah, hang on. The fix wasn't in v0.17.0. Reading back, the fix is in `v0.17.9-pre`. You could update to the latest prerelease using `wails update -pre` or if you really want that version `wails update -version v0.17.9-pre`.
